openedx/core/djangoapps/credit/models.py

```diff
@@ -159,30 +159,18 @@ def get_credit_providers(cls, providers_list=None):
         # This doesn't happen very often, so we would expect a *very* high
         # cache hit rate.
 
+        # Import internally to avoid the circular import.
```

Comment:
Clean this method up. There is no need to return a list of dictionaries. We *should not* be using serializers in `models.py`. All you need is `return CreditProvider.objects.filter(active=True)`.